lms/static/js/instructor_dashboard/student_admin.js

```diff
@@ -553,7 +560,10 @@
                     success: this.clear_errors_then(function() {
                         return alert(fullSuccessMessage);  // eslint-disable-line no-alert
                     }),
-                    error: statusAjaxError(function() {
+                    error: statusAjaxError(function(response) {
+                        if (response.responseText === 'Specified module does not support rescoring.') {
+                            return that.$request_err_grade.text(rescoreUnsupportedMessage);
+                        }
```

Comment:
I found this code confusing too, but tested it manually. When the instructor submits a rescore for a non-rescorable block, this message gets surfaced on the instructor dashboard in red text. On success, there isn't actually a pop-up alert- the 'success' text just gets surfaced in green instead. 🤔